classes/book.rb

```diff
@@ -1 +1,15 @@
 # book class
+class Book
+  attr_accessor :publisher, :cover_state, :name
+
+  def initialize(name, cover_state, publisher)
+    super(publish_date)
+    @name = name
+    @publisher = publisher
+    @cover_state = cover_state
+  end
+
+  def can_be_archived?()
```

Comment:
- This method `can_be_archived?()` is a private method in the UML class diagram. Please implement this